src/h3lib/include/h3api.h

```diff
@@ -129,6 +129,16 @@ struct LinkedGeoPolygon {
     LinkedGeoPolygon *next;
 };
 
+/** @struct CoordIJ
+ * @brief IJ hexagon coordinates
+ *
+ * Each axis is spaced 120 degrees apart.
+ */
+typedef struct {
+    int i;  ///< i component
```

Comment:
Nit: Do we really need these inline comments?

Reply:
@dfellis they parallel the inline comments on components in `CoordIJK`, so consistency would argue for keeping them here (or changing them there). Putting them in `CoordIJK` was a coin-flip in favor of completeness, but I agree they're overkill.